org.doom2d.forever.yml

```diff
@@ -0,0 +1,96 @@
+app-id: org.doom2d.forever
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+build-options:
+  append-path: /usr/lib/sdk/freepascal/bin
+  env:
+    PPC_CONFIG_PATH: /usr/lib/sdk/freepascal/etc
+    FPCDIR: /usr/lib/sdk/freepascal/fpcsrc
+    FPC_DIR: /usr/lib/sdk/freepascal/fpcsrc
+default-branch: stable
+command: Doom2DF
+cleanup:
+  - ['*']
```

Comment:
this basically remove everything that is installed.

Reply:
You are right. I've corrected it.